users/README.md

```diff
@@ -0,0 +1,51 @@
+### Users app
+
+#### Models
+
+##### User
+
+The User model is the one that the Kukkuu app uses as a [default User model](https://docs.djangoproject.com/en/3.2/topics/auth/customizing/#referencing-the-user-model). It extends the AbstractUser model class provided by the _Helusers app_.
+
+> The User model is the one that the Django framework, Helsinki's authentication systems and all the plugins are using for the authentication and authorization.
+
+##### Guardian
+
+The Guardian model is using a 1-to-1 foreignkey to the User-model. It's purpose is to extend the user model so that the Kukkuu's child specific data is separated to another data model, so the User stays more close to the Django's own abstract implementation. Like that, the guardian-user -relationship is more detachable and extendable.
+
+#### GraphQL Schema
+
+[The Users GraphQL schema](./schema.py) contains query fields to read data and mutation fields to update data.
+
+##### Guardians
+
+The `guardians` query gives a list of guardians, but the viweing is limited so that current user can see only himself and only the admin users can see all the guardians.
+
+##### My profile
+
+The `myProfile`-query returns the guardian's data for the current logged in user. The contact information and also the user and the children data are available through the GuardianNode. The data is sensible in that sense.
+
+There is a mutation field `UpdateMyProfileMutation` to update the data that the my profile query provides.
+
+> NOTE: The email field value is not updetable through the `UpdateMyProfileMutation`. There is `UpdateMyEmailMutation` field for that.
+
+###### Update the email
+
+The guardian email field can be updated with the `UpdateMyEmailMutation`. The email field is specia, because we want to verify that the guardian user really has an access to the email box that he updates to their profile.
+
+The `UpdateMyEmailMutation` usage always requires an email verification token. The token can be requested with the `RequestEmailUpdateTokenMutation`. A successfull request to the `RequestEmailUpdateTokenMutation` should send an email to the email box that the user is updating the field value to. A request to the `UpdateMyEmailMutation` can only be successfull with that provided token as an input.
```

Comment:
In all the texts in this PR: successfull -> successful

- https://www.merriam-webster.com/dictionary/successful
- https://word.tips/spelling/successful-vs-successfull/
- https://strategically.co/blog/grammar-tips/how-to-spell-successful/